src/components/Home/overviewData/LatestTreasuryOverview.tsx

```
@@ -227,62 +227,75 @@ const LatestTreasuryOverview = ({ currentTokenPrice, available, priceWeeklyChang
+												{!mythLoading && (
+													<>
+														<div className='no-wrap ml-1 mt-1 flex items-center gap-1 sm:mt-0'>
+															<Image
+																src={'/assets/treasury/myth-icon.svg'}
+																width={15}
+																height={15}
+																alt='icon'
+															/>
+															<span className='text-xs font-medium text-blue-light-high dark:text-blue-dark-high'>{MythBalance} MYTH</span>
+														</div>
+													</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)